message ClearSectionsUplinkPayload {
  required bool clear_buffer = 1 [default = true];
}